File: inventario_frontend/src/models/productEntryLine.ts
export class ProductEntryLineForCreation {
  public quantity: number | undefined = undefined;
  public productId: string | undefined = undefined;
  public poductEntryId: string | undefined = undefined;
}

export class ProductEntryLine extends ProductEntryLineForCreation {
  public id: string | undefined = undefined;
}

File: Inventario/DataAccess/Models/ProductEntryLine.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Data
{
    public class ProductEntryLine: Entity
    {
        public Guid ProductEntryId { get; set; }
        public ProductEntry ProductEntry { get; set; }
        public Guid ProductId { get; set; }
        public Product Product { get; set; }
        public decimal Quantity { get; set; }
    }
}
